packages/preferences/src/components/preferences-modal-section/index.tsx

```diff
@@ -1,4 +1,15 @@
-const Section = ( { description, title, children } ) => (
+/**
+ * External dependencies
+ */
+import type { ReactNode } from 'react';
+
+export type SectionProps = {
+	description: ReactNode;
+	title: ReactNode;
+	children: ReactNode;
+};
```

Comment:
Let us avoid the React import.

```suggestion

export type SectionProps = {
	description: React.ReactNode;
	title: React.ReactNode;
	children: React.ReactNode;
};
```